modules/cas_cache/layer_cache_management.c

```diff
@@ -2344,17 +2346,19 @@ int cache_mngt_exit_instance(const char *cache_name, size_t name_len, int flush)
 	if (flush && !flush_status)
 		BUG_ON(ocf_mngt_cache_is_dirty(cache));
 
-	/* Stop cache device */
+	/* Stop cache device - ignore interrupts */
 	status = _cache_mngt_cache_stop_sync(cache);
-	if (status == -ENOMEM)
-		goto unlock;
-
-	ocf_queue_put(mngt_queue);
+	if (status == -KCAS_ERR_WAITING_INTERRUPTED)
+		printk(KERN_WARNING
+				"An attempt to interrupt stopping cache "
+				"was made. Ignored.\n");
```

Comment:
I wouldn't print "ignored" here. Maybe
Waiting for cache stop interrupted. Stop will finish asynchronously.